GameServer/commands/gmcommands/addbind.cs

```diff
@@ -41,6 +52,8 @@ public void OnCommand(GameClient client, string[] args)
 				}
 				catch (Exception e)
 				{
+					// Message: [Error] {0}

```

Comment:
Displays two messages.
![Screenshot 2023-06-10 020351](https://github.com/OpenDAoC/OpenDAoC-Core/assets/1279721/38427767-8af5-4b2c-a964-eae48b8fef90)

Reply:
Oops, forgot to remove the old message call.